src/sentry/static/sentry/app/views/projectReleases/index.jsx

```diff
@@ -165,20 +182,34 @@ const ProjectReleases = createReactClass({
 
   renderEmpty() {
     const {environment} = this.state;
+    const {organization, project} = this.context;
+    let anyProjectReleases = project.latestRelease;
+
     const message = environment
       ? tct("There don't seem to be any releases in your [env] environment yet", {
           env: environment.displayName,
         })
       : t("There don't seem to be any releases yet.");
-    return (
-      <EmptyStateWarning>
-        <p>{message}</p>
-        <p>
-          <a href={this.getReleaseTrackingUrl()}>
-            {t('Learn how to integrate Release Tracking')}
-          </a>
-        </p>
-      </EmptyStateWarning>
+
+    let EmptyStateComponent = HookOrDefault({
```

Comment:
not quite sure what's going on here, but let's push this and we can refactor later